onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -6,7 +6,7 @@
 import tempfile
 import typing
 import warnings
-from typing import Any, Dict, Final, List, Mapping, Optional, Sequence, Tuple, Union
+from typing import Any, Dict, List, Mapping, Optional, Sequence, Tuple, Union
```

Comment:
```suggestion
from typing import Any, Dict, Final, List, Mapping, Optional, Sequence, Tuple, Union
```

Reply:
strange, not sure why lintrunner removed this. thanks for fix.